### PR TITLE
Update to Coveralls >= 3

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -32,15 +32,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v2.1.4
```
Comment (sallner) [inline .github/workflows/tests.yml:35]: What is the reason to pin this version strictly? I guess updates on the v2 branch should not break the builds.

**Review by sallner — APPROVED** [review]
👍🏾

File: .github/workflows/tests.yml (hunk 1 above)
Comment (icemac) [reply .github/workflows/tests.yml:35]: They where breaking the builds before in a version newer then the specified one. I was not yet able to check if v2 is enough, but at least it seems to work for now.

Comment (sallner) [reply .github/workflows/tests.yml:35]: 👍🏾

Comment (icemac) [reply .github/workflows/tests.yml:35]: Let's try v2 here.

**Pull request merged**